doc/tor.md

```diff
@@ -86,7 +100,7 @@ and open port 8333 on your firewall (or use -upnp).
 If you only want to use Tor to reach onion addresses, but not use it as a proxy
 for normal IPv4/IPv6 communication, use:
 
-	./bitcoin -onion=127.0.0.1:9050 -externalip=57qr3yd1nyntf5k.onion -discover
+	./bitcoin -onion=127.0.0.1:9050 -externalip=57qr3yd1nyntf5k.onion -discover -listen -onlynet=onion -bind=127.0.0.1
```

Comment:
Also for (2) I personally prefer:
```
./bitcoin -proxy=127.0.0.1:9050 -externalip=57qr3yd1nyntf5k.onion -discover -listen -onlynet=onion -bind=127.0.0.1
```
This has the same effect, but in the case that there is a bug causing bitcoind to ignore `onlynet` and try to connect to IPv4/IPv6 anyway, at least it will go through Tor.